File: 백준/Bronze/2480. 주사위 세개/주사위 세개.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int n1 = sc.nextInt();
        int n2 = sc.nextInt();
        int n3 = sc.nextInt();
        if(n1 == n2 && n2 == n3) {
            System.out.println(10000 + n1 * 1000);
        } else if(n1 == n2 || n2 == n3 || n3 == n1) {
            System.out.println(1000 + (n1 == n2 ? n1 : n3) * 100);
        } else {
            System.out.println(Math.max(n1, Math.max(n2, n3)) * 100);
        }
        sc.close();
    }
}

File: 백준/Bronze/2480. 주사위 세개/주사위 세개.cc
#include <stdio.h>

int main()
{
    int a, b, c, m, max;

    scanf("%d %d %d", &a, &b, &c);

    if (a == b && b == c)
        printf("%d", 10000 + a*1000);
    else if (a == b || a == c)
        printf("%d", 1000 + a*100);
    else if (b == c)
        printf("%d", 1000 + b*100);
    else {
        max = a;
        if (max < b)
            max = b;
        if (max < c)
            max = c;
        printf("%d", max * 100);
    }
    
    return 0;
}